build/build.xml

```diff
@@ -413,9 +413,6 @@
 		<build-addon name="tlsdebug" />
 		<build-addon name="viewstate" />
 		<build-addon name="wappalyzer" />
-		<!-- Need to download the drivers before building the webdriver add-ons -->
-		<antcall target="clean-webdrivers"/>
```

Comment:
Yes, thats intentional - I was finding sometimes they werent getting picked up in the build. You can still choose to invoke the 'clean-webdrivers' task if you need to :)